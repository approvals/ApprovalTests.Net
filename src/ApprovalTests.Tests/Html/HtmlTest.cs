using ApprovalTests.Reporters;
using NUnit.Framework;

namespace ApprovalTests.Tests.Html
{
    [TestFixture]
    [UseReporter(typeof(DiffReporter), typeof(FileLauncherReporter))]
    public class HtmlTest
    {
        [Test]
        public static void TestHtml()
        {
            Approvals.VerifyHtml("<html><body><div style='font-family:Broadway;font-size:18px'> Web Page from ApprovalTests</div></body></html>");
        }
    }
}