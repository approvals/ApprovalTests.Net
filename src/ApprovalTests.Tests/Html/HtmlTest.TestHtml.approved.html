<html>
  <body>
    <div style="font-family:Broadway;font-size:18px"> Web Page from ApprovalTests</div>
  </body>
</html>